doc/feature/feature.md

```diff
@@ -1,8 +0,0 @@
-Feature specifications {#FeatureSpecs}
-======================
-
-* @subpage compartmentReport
-* @subpage compartmentReportMPI
-* @subpage morphology11
-* @subpage spikeReport
-* @subpage brainSynapses
```

Comment:
Because it was duplicated in a file with a different name. However I'm going to put it back.